connector/connector_ldap.go

```diff
@@ -210,18 +231,131 @@ func (c *LDAPConnector) Register(mux *http.ServeMux, errorURL url.URL) {
 }
 
 func (c *LDAPConnector) Sync() chan struct{} {
+	go func() {
+		for {
+			alive, killed := c.idp.ldapPool.Ping()
+			if alive > 0 {
+				log.Infof("Connector ID=%v idle_conns=%v", c.id, alive)
+			}
+			if killed > 0 {
+				log.Warningf("Connector ID=%v closed %v dead connections.", c.id, killed)
+			}
+			time.Sleep(c.idp.ldapPool.PoolCheckTimer)
+		}
+	}()
 	return make(chan struct{})
```

Comment:
Thx, I'll have a look at connector_oidc and make a new attempt at making it work.